### DISCUSSION TITLE
Development of app running on background

### DISCUSSION THREAD

**Discussion by JanBN** [zepp-health/zeppos-forum#21] [Q&A] [answered]
hello,

I would like to have an app that reminds me to train my eyes, by looking at distance every hour or so. The app should only vibrate a few times, show some text "look at distance" and then disappear again. Meaning I do not need to interact with the app. I was using the first amazfit bip and reminders did work this way.
I did try to replicate this behavior by todo list but I needed to interact with the notification by clicking on tick or cancel.

So my question is, is this somehow possible on the new amazfit os?
If not, is it possible to create such an app? The important part is that the app would count time in the background, from time to time vibrates and shows notification and then again automatically disappears.

**Comment by liqin-ZeppHealth** [comment, marked as answer]
alarm would be a possible solution to launch your app to display this notification, or new ability of running on background will be available later in new Zepp OS.

**Reply by JanBN** [reply]
Ok thank you for the reply. I have no problem to create such an app, however it must be possible by OS. What do you mean by later. In a few weeks, months, or years ? And would it be supported on current amazfit devices, or only on the next generation.
